Readme.md

```diff
@@ -17,6 +17,13 @@ We are an open community of volunteers without a commercial purpose. We believe
 #### Python Version
 covidX will be run on python *3.7.6* and *3.8.5*
 
+#### Pre-requisite Services
```

Comment:
if you look at .env file and the dependent services, you will realize this is not enough.
Look at the Applications being installed in `settings.py` again; 
In addition to that, We also need ALGOLIA api key, can you think of other dependencies here?

Reply:
I thought that new developers were provided with the Algolia api key by the team leads. So, are you saying I should give them instructions on how to obtain this themselves?

Reply:
no. the objective of this description is for anyone who wants to self-host this repository to understand what all services are needed in order to spin up their own instance.

Reply:
That makes a lot more sense now. So, they will need everything.

Reply:
yes, you need to tell them what services one would need to spin this up